### ISSUE TITLE
Learning without Data: Physics-Informed Neural Networks for Fast Time-Domain Simulation. (arXiv:2106.15987v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#8457]
Learning without Data: Physics-Informed Neural Networks for Fast Time-Domain Simulation. (arXiv:2106.15987v1 [eess.SY])<br>
https://ift.tt/3js6mZv<br>
<p>In order to drastically reduce the heavy computational burden associated with time-domain simulations, this paper introduces a Physics-Informed Neural Network (PINN) to directly learn the solutions of power system dynamics. In contrast to the limitations of classical model order reduction approaches, commonly used to accelerate time-domain simulations, PINNs can universally approximate any continuous function with an arbitrary degree of accuracy. One of the novelties of this paper is that we avoid the need for any training data. We achieve this by incorporating the governing differential equations and an implicit Runge-Kutta (RK) integration scheme directly into the training process of the PINN; through this approach, PINNs can predict the trajectory of a dynamical power system at any discrete time step. The resulting Runge-Kutta-based physics-informed neural networks (RK-PINNs) can yield up to 100 times faster evaluations of the dynamics compared to standard time-domain simulations. We demonstrate the methodology on a single-machine infinite bus system governed by the swing equation. We show that RK-PINNs can accurately and quickly predict the solution trajectories.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
